packages/legend-application-studio/src/components/editor/edit-panel/project-configuration-editor/ProjectConfigurationEditor.tsx

```diff
@@ -105,27 +154,87 @@ const ProjectStructureEditor = observer(
         </div>
 
         <PanelForm>
-          <PanelFormTextField
-            isReadOnly={isReadOnly}
-            value={projectConfig.groupId}
-            name="Group ID"
-            prompt="The domain for artifacts generated as part of the project build
-              pipeline and published to an artifact repository"
-            update={(value: string | undefined): void =>
-              projectConfig.setGroupId(value ?? '')
-            }
-          />
-          <PanelFormTextField
-            isReadOnly={isReadOnly}
-            value={projectConfig.artifactId}
-            name="Artifact ID"
-            prompt="The identifier (within the domain specified by group ID) for
+          <PanelFormSection>
```

Comment:
in terms of UX, what I think makes sense to me is we do the block on `Update Config` action, this is the time when the change happens. If we need people to be really cautious about their changing the `gav` coordinates, we can have a add warning to the input, with potentially link to documentation